### PR TITLE
Document use of sequelize-cli to perform migrations

### PR DESCRIPTION
**Pull request by submitteddenied**

### CODE REVIEW

**Comment by submitteddenied** [comment]
RE: #9

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 ## Setup
 
 - install this plugin: `npm install ah-sequelize-plugin --save`
-- be sure to enable the pluggin within actionhero (`config/plugins.js`)
+- be sure to enable the plugin within actionhero (`config/plugins.js`)
 - you will need to add the sequelize package (`npm install sequelize --save`) to your package.json
 - you will need to add the sequelize-fixtures package (`npm install sequelize-fixtures --save`) to your package.json
 - you will need to add the mysql (or other supported database) package (`npm install mysql --save`) to your package.json
   - there are many options you can pass to sequelize.  You can learn more here: http://sequelize.readthedocs.org/en/latest/api/sequelize/index.html
+- you will need to add the sequelize-cli package (`npm install sequelize-cli --save-dev`) to the `"devDependencies"` of your package.json
```
Comment (evantahler) [inline README.md]: What if you want to migrate production?  I would remove the `--save-dev` here

**Comment by submitteddenied** [comment]
Ping @evantahler I removed the `--save-dev` in the readme. Everything look ok?

**Comment by evantahler** [comment]
great!

(it's a shame github doesn't ping you when code is added to a PR)

**Pull request merged**